new/LIT6458MashafaZ.xml

```diff
@@ -0,0 +1,153 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6458MashafaZ" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መጽሐፈ፡ ዜናቲሆሙ፡ ለአበዊነ፡ ቅዱሳን፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Maṣḥafa zenātihomu la-ʾabawina qǝddusān</title>
+                <title xml:lang="la" xml:id="t2">Quadraginta historiae monachorum</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+              <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="MonasticLiterature"/>
+                    <term key="Translation"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+                <language ident="la">Latin</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MKr" when="2021-06-08">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+          <div type="bibliography">
+            <listBibl type="editions">
+                <bibl><ptr target="bm:Arras1988Quadraginta"/></bibl>
+            </listBibl>
+              <listBibl type="secondary">
+                <bibl><ptr target="bm:Bausi2007Monastic"/></bibl>
+              </listBibl>
+          </div>
+          <div type="edition">
+             <div type="textpart" subtype="incipit" xml:lang="gez">
+                <ab>
+                  ንዌጥን፡ በረድኤተ፡ እግዚአብሔር፡ ወሥነ፡ ሀብቱ፡ ወመርሑ፡ ጽሒፈ፡ መጽሐፈ፡ ዜናቲሆሙ፡ ለአበዊነ፡ ቅዱሳን፡ ንጹሓን፡ ባሕታውያን፡
+                </ab>
```

Comment:
The div with the type incipit should be closed here, otherwise, all the following divs are contained in it. Just move one of the closing tags from the end here.